.github/workflows/release.yml

```diff
@@ -68,6 +68,46 @@ jobs:
     needs: create-release-branch
     env:
       URL: ${{ secrets.RELEASE_NOTIFICATON_URL }}
+      ImTagStr: 'imageTag:.*'
+      e2eStr: 'enabled: true'
+      e2eDisRepl: 'enabled: false'
+      FrontendVer: ${{ github.event.client_payload.slash_command.frontend }}
+      FrontImVer: 'imageTag: ${{ github.event.client_payload.slash_command.frontend }}'
+      BackendVer: ${{ github.event.client_payload.slash_command.backend }}
+      BackImVer: 'imageTag: ${{ github.event.client_payload.slash_command.backend }}'
+      GitApiVer: ${{ github.event.client_payload.slash_command.gitapi }}
+      GitApiImVer: 'imageTag: ${{ github.event.client_payload.slash_command.gitapi }}'
+      DispatcherVer: ${{ github.event.client_payload.slash_command.dispatcher }}
+      DispatcherImVer: 'imageTag: ${{ github.event.client_payload.slash_command.dispatcher }}'
+      AgnVVer: ${{ github.event.client_payload.slash_command.agnosticv }}
+      AnarchyVer: ${{ github.event.client_payload.slash_command.anarchy }}
+      PoolBVer: ${{ github.event.client_payload.slash_command.poolboy }}
+      StatusVer: ${{ github.event.client_payload.slash_command.status }}
+      StatusImVer: 'imageTag: ${{ github.event.client_payload.slash_command.status }}'
+      ## Lodestar Config has only imageTag specified
+      ConfigImVer: 'imageTag: ${{ github.event.client_payload.slash_command.status }}'
+      #Variables for Status page
+      SFrontStr: 'lodestar-frontend:.*'
+      SBackStr: 'lodestar-backend:.*'
+      SGitApiStr: 'lodestar-git-api:.*'
+      SStatusStr: 'lodestar-status:.*'
+      SConfigStr: 'lodestar-config:.*'
+      SDispatchStr: 'resource-dispatcher:.*'
+      SAgnosticVStr: 'agnosticv-operator:.*'
+      SAnarchyStr: 'anarchy:.*'
+      SPoolBStr: 'poolboy:.*'
+      SLodeStarStr: 'lodestar:.*'
+      SFrontVer: 'lodestar-frontend:  ${{ github.event.client_payload.slash_command.frontend }}'
+      SBackVer: 'lodestar-backend: ${{ github.event.client_payload.slash_command.backend }}'
+      SGitApiVer: 'lodestar-git-api: ${{ github.event.client_payload.slash_command.gitapi }}'
+      SStatusVer: 'lodestar-status: ${{ github.event.client_payload.slash_command.status }}'
+      SConfigVer: 'lodestar-config: '    #### To be filled in
+      SDispatchVer: 'resource-dispatcher: ${{ github.event.client_payload.slash_command.dispatcher }}'
+      SAgnosticVVer: 'agnosticv-operator: ${{ github.event.client_payload.slash_command.agnosticv }}'
+      SAnarchyVer: 'anarchy: ${{ github.event.client_payload.slash_command.anarchy }}'
+      SPoolBVer: 'poolboy: ${{ github.event.client_payload.slash_command.poolboy }}'
+      SLodeStarVer: 'lodestar: ${{ github.event.client_payload.slash_command.lodestar }}''
```

Comment:
LodeStar version is obtained from the issue title - not by the slash_command - as seen here:
https://github.com/rht-labs/lodestar-deployment/blob/master/.github/workflows/release.yml#L134

Reply:
I reverted the change, now Lodestar version is populated with issue title